package.json

```diff
@@ -112,5 +115,12 @@
     "redux": "^3.7.2",
     "redux-form-validators": "^2.4.3",
     "redux-mock-store": "^1.5.1"
+  },
+  "release": {
+    "analyzeCommits": [
```

Comment:
Is this specific to our own commit analyzer?

Because, looking at https://github.com/semantic-release/commit-analyzer#options , it looks like analyzeCommits is always an object, not an array..

Reply:
Their documentation sucks, it can either be object or array of objects.